bsp/src/org/jetbrains/bsp/project/importing/bspConfigSteps.scala

```diff
@@ -103,9 +103,6 @@ object bspConfigSteps {
           builder.setPreImportConfig(NoPreImport)
           // server config to be set in next step
           SbtConfigSetup(workspace)
-        case bspConfigSteps.MillSetup =>
-          builder.setPreImportConfig(NoPreImport)
-          MillConfigSetup(workspace)
```

Comment:
Taking this case out causes a compile error since we have fatal warnings now. Should it just be an empty case, or the `MillSetup` object removed?
```
[error] /Users/jast/workspace/scalaUltimate/community/bsp/src/org/jetbrains/bsp/project/importing/bspConfigSteps.scala:89:12: match may not be exhaustive.
[error] It would fail on the following input: MillSetup
[error]     } else configSetup match {
```

Reply:
I pushed a new implementation which is much better actually.